src/dotnet/Audio.Service/Processing/OpenAudioSegment.cs

```diff
@@ -47,14 +47,6 @@ public void SetRecordedAt(Moment? recordedAt)
                 Index, StreamId);
     }
 
-    public void SetAudibleDuration(TimeSpan audibleDuration)
```

Comment:
Why it's deleted? I.e. we are supposed to it, and if it wasn't used, we should investigate why. @alexis-kochetov is this because of VAD?

Reply:
added todo comment for @alexis-kochetov